### ISSUE TITLE
Layout/TrailingBlankLines not reported anymore (it seems)

### ISSUE THREAD

**Issue by mvangoor** [chef/cookstyle#816]
# Version:

Cookstyle 7.3.11
  * RuboCop 1.5.2

# Environment:

chef workstation image 20.12.205

# Scenario:

Current version of Cookstyle is not reporting trailing blank lines as an error.

Example older cookstyle (using chefdk):
```
Inspecting 29 files
.........................C...
Offenses:
recipes/acls.rb:70:1: C: Layout/TrailingBlankLines: 3 trailing blank lines detected.
29 files inspected, 1 offense detected
```

# Steps to Reproduce:

Create a file with trailing blank lines and run cookstyle, observe it is not reporting an error

# Expected Result:

Reports correctly that there are trailing blank lines per <https://github.com/chef/cookstyle/blob/master/config/cookstyle.yml#L2327>

# Actual Result:

File is accepted incorrectly

**Comment by tas50** [comment]
The cop has been renamed to TrailingEmptyLines. I'll fix that.
